skopt/optimizer/optimizer.py

```diff
@@ -233,46 +239,62 @@ def tell(self, x, y, fit=True):
                 warnings.simplefilter("ignore")
                 est.fit(self.space.transform(self.Xi), self.yi)
 
+            if hasattr(self, "next_xs_") and self.acq_func == "gp_hedge":
+                self.gains_ -= est.predict(np.vstack(self.next_xs_))
```

Comment:
I've set `eta` to be 1.0 over here. Cannot find what value it is being set to in the paper?